indexed-store/index.js

```diff
@@ -1,5 +1,8 @@
 let { isFirstOlder } = require('@logux/core')
 
+if ( !global )  // global is undefined in the Browser. NF 7/05/2020
+    global = window
```

Comment:
This line will set `global` at the global level.

Can we do it in another way? For instance, we can replace `if (global.document && …)` to `if (typeof document !== 'undefined' && …)`

Reply:
Yes sure. I'm using es6 modules so it shouldn't have been an issue for me.